exec/java-exec/src/test/java/org/apache/drill/exec/client/DrillClientSystemTest.java

```diff
@@ -51,7 +51,7 @@ public void tearDownTest() {
   @Test
   public void testSubmitPlanSingleNode() throws Exception {
     startCluster(1);
-    DrillClient client = new DrillClient();
+    DrillClient client = DrillClient.newBuilder().build();
```

Comment:
Let's keep the default constructor to be the same as the default builder. No reason to force a builder when there are no properties.